### PR TITLE
Added selection sort code and the corresponding tests.

### LINKED ISSUE #14: Implement selection sort
Implement a selection sort.

### PR DESCRIPTION
**Pull request by EliasPinno**
Closes #14, and related to #15. This also adds the test folder to store future tests.

### CODE REVIEW

File: tests/selection_sort_tests.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import os
+import sys
+
+sys.path.append(os.path.join(os.path.dirname(__file__), ".."))
+
+# Merge Sort Tests
+import unittest
+from selection_sort import selection_sort
+
+class TestMergeSort(unittest.TestCase):
```
Comment (IvanIsCoding) [inline tests/selection_sort_tests.py]: Nice testing merge sort buddy

Comment (EliasPinno) [reply tests/selection_sort_tests.py]: Sorry! Fixed immediately.

**Review by QuinnMarshal — COMMENTED** [review]
LGTM!
Lets see what Kev thinks

File: tests/selection_sort_tests.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# Import hacks for compatibility
+import os
+import sys
+
+sys.path.append(os.path.join(os.path.dirname(__file__), ".."))
```
Comment (keyvankhademi) [inline tests/selection_sort_tests.py:5]: Why is this here?

**Review by keyvankhademi — APPROVED** [review]

File: tests/selection_sort_tests.py (hunk 2 above)
Comment (EliasPinno) [reply tests/selection_sort_tests.py:5]: Tests are in a test folder, nested one layer deep from the actual algorithms. This was a way @IvanIsCoding showed me to get the parent in the most safe way, irrespective of where you are running the program from.
